packages/webvh/src/cryptosuites/eddsa-jcs-2022.ts

```diff
@@ -0,0 +1,141 @@
+import { Buffer } from 'buffer'
+import { createPublicKey, verify } from 'crypto'
```

Comment:
You should not use crypto directly, we have a `kms` API that should handle all crypto operations (there's a lot of places in the codebase where you can see it being used)